lessons/04-client-jquery-ajax/README.md

```diff
@@ -554,52 +381,16 @@ Last class we discussed folder structure, so as a refresher, draw out the folder
   * An `Add` button should allow the user to specify the name and price of a new ingredient which should appear on the page without requiring a refresh.
-  * Out-of-Stock button will tell the server to label the ingredient as disabled.
-  The ingredient should be removed from the current page without refreshing.
- (Optional: make it toggleable to "add" more of the ingredient.
-In this case, do not remove the ingredient from the page, but make note through words or style that it is unavailable.)
-  * Edit button allows the user to submit a new name or price for the ingredient
-  which the server will update. The edits should change the ingredient list without refreshing.
-* `/order` =>
-  * Shows a form which allows customers to create a new burger.
-  * There should be a [checklist](http://www.w3schools.com/tags/att_input_type.asp) of all ingredients and their price.
-  * Out-of-stock ingredients should have a disabled checkbox (`<input type="checkbox" disabled>`)
-  * There should be a Submit button that will send the server the new order without refreshing the page.
-  * You may want to refer to the Mongo reading on [Referencing vs. Embedding](https://github.com/olinjs/olinjs/blob/master/lessons/03-express-templates-mongo/README.md) as you think about how you will store your data. 
-  You should give your customer a nice congratulatory message for completing their order
-  (maybe a free cat picture since you're so good at that?!)
-  * A running counter of total cost: Should update whenever a new ingredient is added or removed.
-* `/kitchen` =>
-  * Shows a list of all pending orders.
-  * A `completed` button beside each order that tells the server the order is complete.
-  Clicking this should remove the order from the list of orders without refreshing the page.
-
-When you're finished, fill out the [Homework 5 Submission Survey] (https://docs.google.com/forms/d/1MsGf_gNkHJBjBEzQoaUXvmpc3Nljk838FeBXlJ5hH9c/viewform?usp=send_form). 
-
-#### Preclass Reading and Exercise
-- Read the [Class 6 README](https://github.com/olinjs/olinjs/tree/master/lessons/06-css-development-style) before class 6 on Friday 2/5/16.
-- Send an email to [olinjs16@gmail.com](olinjs16@gmail.com) with the subject line "Preclass 6" telling us about...
-    - Something in the reading you felt confident about and easily grasped
-    - Something in the reading you're confused about or want to know more about
+  * Out-of-Stock button will tell the server to label the ingredient as disabled. The ingredient should be removed from the current page without refreshing. (Optional: make it toggleable to "add" more of the ingredient. In this case, do not remove the ingredient from the page, but make note through words or style that it is unavailable.)
+  * Edit button allows the user to submit a new name or price for the ingredient which the server will update. The edits should change the ingredient list without refreshing.
+
```

Comment:
Can we say something here like: This homework is the first part of a bigger application. If you'd like to see the other endpoints that the application will need, look [here] ().